pkg/service/backup/service_backup_integration_test.go

```diff
@@ -2496,3 +2496,86 @@ func TestBackupAlternatorIntegration(t *testing.T) {
 		}
 	}
 }
+
+func TestBackupViews(t *testing.T) {
+	const (
+		testBucket   = "backuptest-views"
+		testKeyspace = "backuptest_views"
+		testTable    = "base_table"
+		testMV       = "mv_table"
+		testSI       = "si_table"
+	)
+
+	location := s3Location(testBucket)
+	config := defaultConfig()
+
+	var (
+		session        = CreateScyllaManagerDBSession(t)
+		h              = newBackupTestHelper(t, session, config, location, nil)
+		ctx            = context.Background()
+		clusterSession = CreateSessionAndDropAllKeyspaces(t, h.Client)
+	)
+
+	WriteData(t, clusterSession, testKeyspace, 1, testTable)
+	CreateMaterializedView(t, clusterSession, testKeyspace, testTable, testMV)
+	CreateSecondaryIndex(t, clusterSession, testKeyspace, testTable, testSI)
+
+	Print("When: prepare target")
+	props := map[string]any{
+		"location": []Location{location},
+		"keyspace": []string{testKeyspace},
+	}
+	rawProps, err := json.Marshal(props)
+	if err != nil {
+		t.Fatal(errors.Wrap(err, "create raw properties"))
+	}
+	target, err := h.service.GetTarget(ctx, h.ClusterID, rawProps)
+	if err != nil {
+		t.Fatal(errors.Wrap(err, "create target"))
+	}
+
+	Print("And: validate target")
+	expected := []backup.Unit{
+		{
+			Keyspace: testKeyspace,
+			Tables:   []string{testTable},
+		},
+	}
+	if diff := cmp.Diff(target.Units, expected,
+		cmpopts.IgnoreFields(backup.Unit{}, "AllTables"),
+		cmpopts.IgnoreSliceElements(func(u backup.Unit) bool { return u.Keyspace == "system_schema" }),
+	); diff != "" {
+		t.Fatal("target units are not as expected", diff)
+	}
+
+	Print("When: run backup")
+	if err := h.service.Backup(ctx, h.ClusterID, h.TaskID, h.RunID, target); err != nil {
+		t.Fatal(errors.Wrap(err, "run backup"))
+	}
+
+	Print("When: list backup files")
+	filesInfo, err := h.service.ListFiles(ctx, h.ClusterID, []Location{location}, backup.ListFilter{ClusterID: h.ClusterID})
+	if err != nil {
+		t.Fatal(errors.Wrap(err, "list backup files"))
+	}
+
+	Print("And: validate no views in backup files")
+	var foundTable int
+	for _, fi := range filesInfo {
+		for _, fm := range fi.Files {
+			if fm.Keyspace == testKeyspace {
+				if fm.Table == testTable {
+					foundTable++
+				}
+				if fm.Table == testSI || fm.Table == testMV {
+					t.Fatal("Found view in backup files")
+				}
+			}
+		}
+	}
+
+	Print("And: validate base table in backup files")
+	if foundTable != len(ManagedClusterHosts()) {
+		t.Fatal("Expected all hosts to back up base table")
+	}
+}
```

Comment:
Take a look at this very nice explanation of what given-when-then is -> https://martinfowler.com/bliki/GivenWhenThen.html

Your prints doesn't follow this pattern.
Please add print with Given -> current state
When -> What action you did
Then -> What is the expected state

Reply:
done